### ISSUE TITLE
No organizations visible in Meli

### ISSUE THREAD

**Issue by klavierworks** [getmeli/meli#226]
When using Github authentication, Meli does not seem to return/cannot find any organizations attached to my user.

OAuth completes fine and returns me back to Meli without any errors. Also correctly requests permissions for organization:
![Screenshot 2021-05-06 at 11 52 15](https://user-images.githubusercontent.com/5897209/117280052-849f6a00-ae62-11eb-9f5f-dbd7aa02cea7.png)

![Screenshot 2021-05-06 at 11 52 21](https://user-images.githubusercontent.com/5897209/117280070-8832f100-ae62-11eb-86db-c625aaa667a1.png)

However, when in Meli, the organizations list is empty:
![Screenshot 2021-05-06 at 11 52 31](https://user-images.githubusercontent.com/5897209/117280140-95e87680-ae62-11eb-8beb-16767f8601c2.png)

Checking Dev Tools and the GET request is completing fine, code 200, but there is an empty response:
![Screenshot 2021-05-06 at 11 52 44](https://user-images.githubusercontent.com/5897209/117280218-aac50a00-ae62-11eb-9d3f-32da51b1133e.png)

For reference, I was able to use Meli fine when using standard authentication, but Github integration would be preferable.

**Comment by gempain** [comment]
Hi there. Each auth method creates new users that aren't initially linked to an organization. To see the organization you initially created with the in-memory auth, you will need to:
1. connect using the in-memory auth
2. invite a new user, using which ever email address is necessary
3. connect to Meli using your Github account
4. Click the invite link sent to you by email and accept the invite

You MUST be logged in to have access to the actual invitation. We know this is a bit odd, but its due to how our data model works in the background. It's something we have to improve. Was previously mentioned in #224.

Let me know if this helps.

**Comment by klavierworks** [comment]
I think perhaps I'm misunderstanding the organization concept, then – I was under the impression these would match the organizations available in Github? Is that not the case?

**Comment by gempain** [comment]
No, it is not matching the organization from Github. You have to create an organization in Meli. The only feature Meli has which uses Github orgs is that you can restrict access to users that only belong to specific orgs (so that users from other orgs cannot log into your Meli instance).
